runtime-acceptance-tests/src/main/java/org/activiti/cloud/qa/steps/RuntimeBundleSteps.java

```diff
@@ -77,15 +77,16 @@ public void checkServicesHealth() {
     }
 
     @Step
-    public CloudProcessInstance startProcess() {
-        return this.startProcess(SIMPLE_PROCESS_INSTANCE_DEFINITION_KEY);
+    public CloudProcessInstance startProcess(Map<String, Object> variables) {
+        return this.startProcess(SIMPLE_PROCESS_INSTANCE_DEFINITION_KEY, variables);
     }
 
     @Step
-    public CloudProcessInstance startProcess(String process) {
+    public CloudProcessInstance startProcess(String process, Map<String, Object> variables) {
```

Comment:
@ffazzini it will be great to have the overloaded method, so we don't need to send null when we don't want to set variables. And avoid all the further changes.

Reply:
ok